api/assemblyline/assemblyline.py

```diff
@@ -131,6 +134,35 @@ def poll_for_results():
     return True
 
 
+def resubmit_stale_scans():
+    session = next(get_db_session())
+    current_time = datetime.utcnow()
+    one_day_ago = current_time - timedelta(days=1)
+    try:
+        scans_in_progress = (
+            session.query(Scan)
+            .filter(
+                and_(
+                    Scan.verdict == ScanVerdicts.IN_PROGRESS.value,
+                    Scan.submitted < one_day_ago,
```

Comment:
Shouldn't this be `Scan.submitted > one_day_ago`?

Reply:
it's misleading because it's a comparison of two dates. Let me know if you have any suggestions on how to reword `one_day_ago` so that it's more clear.

e.g. `2021-11-30` < `2021-12-01`

Reply:
Ah, my bad.  That makes sense.  I had been thinking in my head it was this:

```
Scan.submitted > one_day_old
```